geocat/comp/climatologies.py

```diff
@@ -518,24 +561,54 @@ def climatology_average(
     # Retrieve calendar name
     calendar = _infer_calendar_name(dset[time_dim])
 
+    attrs = {}
+    if keep_attrs or keep_attrs is None:
+        attrs = dset.attrs
+
```

Comment:
```suggestion
    attrs = {}
    if keep_attrs in {True, None}:
        attrs = dset.attrs
```
As discussed, we may want to implement a more universal helper function to handle this, but for now, I think this change will reduce unintended results.